examples/visualization/plot_3d_to_2d.py

```diff
@@ -0,0 +1,123 @@
+
+subjects_dir = mne.datasets.sample.data_path() + '/subjects'
+path_data = mne.datasets.misc.data_path() + '/ecog/sample_ecog.mat'
+
+# We've already clicked and exported
+layout_path = op.join(op.dirname(mne.__file__), 'data', 'image')
+layout_name = 'custom_layout.lout'
+
+###############################################################################
+# Load data
+# ---------
+#
+# First we'll load a sample ECoG dataset which we'll use for generating
+# a 2D snapshot.
+
+mat = loadmat(path_data)
+ch_names = mat['ch_names'].tolist()
+elec = mat['elec']
+dig_ch_pos = dict(zip(ch_names, elec))
+mon = mne.channels.DigMontage(dig_ch_pos=dig_ch_pos)
+info = mne.create_info(ch_names, 1000., 'ecog', montage=mon)
+print('Created %s channel positions' % len(ch_names))
+
+###############################################################################
+# Project 3D electrodes to a 2D snapshot
+# --------------------------------------
+#
+# Because we have the 3D location of each electrode, we can use the
+# :func:`mne.viz.snapshot_brain_montage` function to return a 2D image along
+# with the electrode positions on that image. We use this in conjunction with
+# :func:`mne.viz.plot_trans`, which visualizes electrode positions.
+
+fig = plot_trans(info, trans=None, subject='sample', subjects_dir=subjects_dir)
+mlab.view(200, 70)
+xy, im = snapshot_brain_montage(fig, mon)
+
+# Convert from a dictionary to array to plot
+xy_pts = np.vstack(xy[ch] for ch in info['ch_names'])
+
+# Define an arbitrary "activity" pattern for viz
+activity = np.linspace(100, 200, xy_pts.shape[0])
+
+# This allows us to use matplotlib to create arbitrary 2d scatterplots
+fig2, ax = plt.subplots(figsize=(10, 10))
+ax.imshow(im)
+ax.scatter(*xy_pts.T, c=activity, s=200, cmap='coolwarm')
+ax.set_axis_off()
+# fig2.savefig('./brain.png', bbox_inches='tight')  # For ClickableImage
+
+###############################################################################
+# Manually creating 2D electrode positions
+# ----------------------------------------
+#
+# If we don't have the 3D electrode positions then we can still create a
+# 2D representation of the electrodes. Assuming that you can see the electrodes
+# on the 2D image, we can use :class:`mne.viz.ClickableImage` to open the image
+# interactively. You can click points on the image and the x/y coordinate will
+# be stored.
+#
+# We'll open an image file, then use ClickableImage to
+# return 2D locations of mouse clicks (or load a file already created).
+# Then, we use generate_2d_layout to turn those xy positions into a layout
+# for use with plotting topo maps. In this way, you can take arbitrary xy
+# positions and turn them into a plottable layout.
+
+
+# This code opens the image so you can click on it. Commented out
+# because we've stored the clicks as a layout file already.
+
+# # The click coordinates are stored as a list of tuples
+# im = plt.imread('./brain.png')
+# click = ClickableImage(im)
+# click.plot_clicks()
```

Comment:
I tried to comment out

im = plt.imread('./brain.png')
click = ClickableImage(im)
click.plot_clicks()

but I get 

ValueError: not enough values to unpack (expected 2, got 0)

I need I demo I guess...